src/typecheck.cpp

```diff
@@ -226,14 +226,18 @@ void typecheck(TypedAST::TernaryExpression* ast, Frontend::CompileTimeEnvironmen
 }
 
 void typecheck(TypedAST::RecordAccessExpression* ast, Frontend::CompileTimeEnvironment& env) {
+	typecheck(ast->m_record.get(), env);
```

Comment:
Is this function defined somewhere?

Reply:
`m_record.get()` is a `TypedAST*`.

`typecheck(TypedAST*, CompileTimeEnvironment&)` is the very last definition in this file.